Makefile

```diff
@@ -44,9 +48,16 @@ uninstall:
 clean:
 	rm rhc_worker_playbook/constants.py
 	rm scripts/rhc-worker-playbook.worker
+	rm rhc-worker-playbook.spec
 
 .PHONY: tarball
 tarball: dist/$(PKGNAME)-$(PKGVER).tar.gz
 
 dist/$(PKGNAME)-$(PKGVER).tar.gz:
 	$(PYTHON) setup.py sdist
+
```

Comment:
Can you please add here `.FORCE: rhc-worker-playbook.spec` here? If it was added, then `Makefile` can be removed from line 59.

Reply:
The syntax for `FORCE` is not what I was expecting.

```
rhc-worker-playbook.spec: rhc-worker-playbook.spec.in FORCE
    sed ...

FORCE:
```

This is not as elegant as using .PHONY, so we're better off using it.